pipelines/jobs/configurations/jdk22_pipeline_config.groovy

```diff
@@ -4,13 +4,23 @@ class Config22 {
         x64Mac    : [
                 os                  : 'mac',
                 arch                : 'x64',
-                additionalNodeLabels: 'xcode15.0.1',
+                additionalNodeLabels: [
+                        openj9      : 'hw.arch.x86 && sw.os.mac.10_15',
+                        temurin     : 'xcode15.0.1'
+                ],
                 additionalTestLabels: [
                         openj9      : '!sw.os.mac.10_11'
```

Comment:
This should be updated to 10_15. We don't have any 10.11 anymore and we've seen on OpenJ9 builds that 22 isn't supported on 10.15. OpenJ9 does still compile on 10.15 though. We need to get the xcode issue sorted too as that will help determine if/what the compile can move onto.

Reply:
Right. But we also don't want 10.15 for jdk22. And we don't have any 10.11 anymore.

Reply:
Got it. Fixed.